gulpfile.js/tasks/css.js

```diff
@@ -16,11 +18,12 @@ var paths = {
 
 var cssTask = function () {
   return gulp.src(paths.src)
-    .pipe(sourcemaps.init())
+    .pipe(gulpif(!global.production, sourcemaps.init()))
     .pipe(sass(config.tasks.css.sass))
     .on('error', handleErrors)
     .pipe(autoprefixer(config.tasks.css.autoprefixer))
-    .pipe(sourcemaps.write())
+    .pipe(gulpif(global.production, cssnano({autoprefixer: false})))
```

Comment:
css-min was throwing a deprecation warning saying to use this instead. But yeah, looks like it's pretty cool! http://cssnano.co/optimisations/

It's just a bunch of postcss rolled together.